packages/wallet/src/redux/protocols/funding/player-a/states.ts

```diff
@@ -36,7 +37,7 @@ export interface WaitForStrategyResponse extends BaseState {
 
 export interface WaitForFunding extends BaseState {
   type: typeof WAIT_FOR_FUNDING;
-  fundingState: 'funding state';
+  fundingState: IndirectFundingState;
```

Comment:
Should terminal funding states be included in the type of `fundingState`?

Reply:
I think you're right that we don't need to include terminal states. I've updated this. We should probably be consistent and only use non-terminal states for other sub protocols as well.